datafusion/sql/tests/sql_integration.rs

```diff
@@ -571,10 +571,6 @@ Dml: op=[Insert Into] table=[test_decimal]
     "INSERT INTO person (id, first_name, last_name) VALUES ($1, $2, $3, $4)",
     "Error during planning: Placeholder $4 refers to a non existent column"
 )]
-#[case::placeholder_type_unresolved(
```

Comment:
What happens now in this case? Does it pass successfully?

Reply:
Right, this PR is intentionally about allowing the creation of `LogicalPlan`'s with placeholders. Previously, `LogicalPlan` creation would fail when any placeholder is part of the schema. Given that all this test is doing is creating LogicalPlan's, this example now passes. I thought about moving it to a new test that explicitly shows it failing at the optimization step but then felt that that case was already covered by the other test updates.